lib/web_ui/lib/src/engine/web_experiments.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+part of engine;
+
+const MethodCodec codec = JSONMethodCodec();
+
+final WebExperiments webExperiments = WebExperiments._();
+
+/// A bag of all experiment flags in the web engine.
+///
+/// This class also handles platform messages that can be sent to enable/disable
+/// certain experiments at runtime without the need to access engine internals.
+class WebExperiments {
+  WebExperiments._();
+
+  /// Experiment flag for using the Skia-based rendering backend.
+  bool get useSkia => _useSkia ?? false;
```

Comment:
While you're here, could you please rename this to `useCanvasKit`? We're consolidating on the name "CanvasKit". Keep the old `FLUTTER_WEB_USE_SKIA` flag for now. Changing that would be a breaking change for the framework tests.